adapt/entity_tagger.py

```diff
@@ -9,7 +9,7 @@ class EntityTagger(object):
     Known Entity Tagger
     Given an index of known entities, can efficiently search for those entities within a provided utterance.
     """
-    def __init__(self, trie, tokenizer, regex_entities=[], max_tokens=20):
+    def __init__(self, trie, tokenizer, regex_entities=None, max_tokens=20):
         self.trie = trie
         self.tokenizer = tokenizer
         self.max_tokens = max_tokens
```

Comment:
You should also deal with the regex_entities=None right here, not later.  So:

if regex_entities is None:
    self.regex_entities = []
else:
    self.regex_entities = regex_entities